qgcimages.qrc

```diff
@@ -196,5 +196,8 @@
         <file alias="Yield.svg">src/ui/toolbar/Images/Yield.svg</file>
         <file alias="ZoomMinus.svg">src/FlightMap/Images/ZoomMinus.svg</file>
         <file alias="ZoomPlus.svg">src/FlightMap/Images/ZoomPlus.svg</file>
+        <file alias="Grid.svg">custom/res/Images/Grid.svg</file>
+        <file alias="grid+line.svg">custom/res/Images/grid+line.svg</file>
```

Comment:
grid+line.svg Change the file name. It should not use + in the file name. Suggestion: GridLine.svg

Reply:
ok

Reply:
done